src/Commands/LangDiffVendorCommand.php

```
@@ -0,0 +1,110 @@
+        return lang_path($relativePath);
+    }
+
+    protected function getLocalFiles(): array
+    {
+        $langPath = lang_path();
+
+        if (! is_dir($langPath)) {
+            return [];
+        }
+
+        $fs = app(Filesystem::class);
```

PHPMD (2.15.0)

Avoid variables with short names like $fs. Configured minimum length is 3. (undefined)

(ShortVariable)